lab_hm_Lambda/src/main/java/fourstask/Main.java

```diff
@@ -0,0 +1,33 @@
+package fourstask;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Main {
+    private static final Logger log = LogManager.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        List<String> list = new ArrayList<>(new Text().getList());
+        log.info(list + " our list of String");
+
+        List<String> list1 = list.stream().flatMap(e -> Stream.of(e.split(" "))).collect(Collectors.toList());

```

Comment:
try to avoid long inlined streams and write operations/ group of operations from new line, this one is not too long but anyway  
like list.stream()
              .flatMap(e -> Stream.of(e.split(" ")))
                  .collect(Collectors.toList())